elasticapm/transport/base.py

```diff
@@ -86,6 +87,7 @@ def __init__(
         self._closed = False
         self._processors = processors if processors is not None else []
         super(Transport, self).__init__()
+        self.priority = sys.maxsize  # ensure that the transport thread is always started/stopped last
```

Comment:
Does that correspond to the native thread's priority? If so, could that impact CPU scheduling in a way that takes away precedence from application threads? If not, maybe choose a different name for the property to avoid these associations. Maybe something like `order`.